composer.json

```diff
@@ -50,6 +51,7 @@
             "src/Php70/bootstrap.php",
             "src/Php71/bootstrap.php",
             "src/Php72/bootstrap.php",
+            "src/Php73/bootstrap.php",
```

Comment:
the branch-alias should be updated to 1.8-dev for all composer.json files of the repo